src/DatePicker/DatePicker.js

```diff
@@ -150,11 +151,6 @@ class DatePicker extends React.Component {
     value: React.PropTypes.any,
 
     /**
-     * Creates a ValueLink with the value of date picker.
-     */
-    valueLink: React.PropTypes.object,
-
-    /**
      * Wordings used inside the button of the dialog.
      */
     wordings: deprecated(React.PropTypes.object, 'Instead, use `cancelLabel` and `okLabel`.'),
```

Comment:
`wordings` was deprecated for 0.15.0. We can remove it in 0.16.0. (I know, I know! 😄)